Colours_And_Fonts_style_v1.css

```diff
@@ -95,7 +161,7 @@ padding-top: 7px;
   
 /* Hover Colours*/
 a:link {color:#FFFFFF;} /* unvisited link */
-a:visited {color:#000000;} /* visited link */
+a:visited {color:#FFFFFF;} /* visited link */

```

Comment:
I changed the hover colours from black to white so people can see it more cearly